iaso/models/org_unit.py

```diff
@@ -548,36 +552,9 @@ def source_path(self):
     def get_reference_instances_details_for_api(self) -> list:
         return [instance.as_full_model() for instance in self.reference_instances.all()]
 
-    def get_extra_fields(self):
-        from iaso.models import Account
-        from iaso.models.data_store import JsonDataStore
-
-        try:
-            datastore = self.jsondatastore_set.get(
-                account=Account.objects.filter(default_version=self.version).first(),
-                slug="extra_fields",
-            )
-            return datastore.content
-        except JsonDataStore.DoesNotExist:
-            return {}
-
-    def set_extra_fields(self, content):
-        from iaso.models import Account
-        from iaso.models.data_store import JsonDataStore
-
-        try:
-            datastore = self.jsondatastore_set.get(
-                account=Account.objects.filter(default_version=self.version).first(),
-                slug="extra_fields",
-            )
-            datastore.content = {**datastore.content, **content}
-            datastore.save()
-        except JsonDataStore.DoesNotExist:
-            self.jsondatastore_set.create(
-                account=Account.objects.filter(default_version=self.version).first(),
-                slug="extra_fields",
-                content=content,
-            )
+    def set_extra_fields(self, fields):
+        self.extra_fields = {**self.extra_fields, **fields}
```

Comment:
```suggestion
        self.extra_fields = self.extra_fields | fields
```

Purely stylistic suggestion: this should work in Python 3.9 ([https://stackoverflow.com/a/26853961](https://stackoverflow.com/a/26853961)).

Reply:
Ow nice I didn't know that! I like that syntax :+1: